src/state/KindeProvider.tsx

```
@@ -141,6 +151,30 @@ export const KindeProvider = ({
     return;
   }, [audience, scope, clientId, domain, redirectUri, logoutUri]);
 
+  const onRefresh = useCallback(
+    (data: RefreshTokenResult): void => {
+      if (mergedCallbacks.onEvent) {
+        mergedCallbacks.onEvent(AuthEvent.tokenRefreshed, data, contextValue);
+      }
+    },
+    [mergedCallbacks, contextValue],
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)